### PR TITLE
Cirrus: enable Fedora 36 aarch64 tasks on EC2

### PR DESCRIPTION
**Pull request by lsm5**
Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

### CODE REVIEW

File: .cirrus.yml
Comment (cevich) [inline .cirrus.yml:232]: Oh no, this is wrong.  The meta_task runs inside a specific container image.

Comment (lsm5) [reply .cirrus.yml:232]: ah ok

Comment (lsm5) [reply .cirrus.yml]: done.

File: Makefile
Comment (cevich) [inline Makefile:81]: Are you sure about this?  My understanding (could be wrong) is: `aarch64` means ARMv8 64-bit, and `arm` means ARMv7 32-bit.  My knowledge is based on playing around with Fedora on Raspbery-pi's a few years ago :wink:

**Review by cevich — APPROVED** [review]
This looks good.  You kept it close to the aardvark work which helps.  Just a minor nit and a question.  Otherwise LGTM assuming the tests are also green.

File: Makefile
Comment (cevich) [reply Makefile:81]: Raspbery-Pi comment Clarification: Meaning my knowledge is old, possibly irreverent, and should be discarded quickly upon the slightest whiff of being incorrect :smile:

Comment (lsm5) [reply Makefile:81]: yes, so this change is expected, right? We don't need aarch64 in the cross task anymore. We only care about the armv7 one unless we wanna send that off to ec2's armv7 instances?

File: .cirrus.yml
Comment (lsm5) [reply .cirrus.yml]: done. PTAL

File: Makefile
Comment (cevich) [reply Makefile:81]: @flouthoc opinions?

**Comment by lsm5** [comment]
@cevich lemme know if you got any ideas about the success task and CIRRUS_BUILD_ID. Else, maybe it's worthwhile to just change across the board to use rpms from rhcontainerbot/podman-next and get rid of the binary.zip part right away. I was hoping to delay that a bit.

**Comment by lsm5** [comment]
@cevich Made some more changes to get rid of binary.zip in success task. If you're cool with the recent changes I can squash commits and re-push.

File: .cirrus.yml
Comment (cevich) [inline .cirrus.yml:220]: Please leave this, it ensures we store the build artifacts when Cirrus runs on the branch, but only after all tasks are successful.

I'm not opposed to using your copr repos during setup of various repos (instead of downloading this zip) but I still want to produce the zip in case it's needed for some reason later.

File: contrib/cirrus/setup.sh
Comment (cevich) [inline contrib/cirrus/setup.sh:23]: This is fine, but please check c/automation_images if `make`, `automake`, `gcc`, `gcc-c++`, and `kernel-devel` are already installed in the fedora-netavark images.  If so, we don't need to install them here, if not please open a PR to add them over there.

(It's a relatively minor thing, so no need to actually incorporate new images)

File: contrib/cirrus/setup.sh
Comment (cevich) [inline contrib/cirrus/setup.sh:26]: I think this will fail if the file doesn't exist: Use the `-f` argument please.  Also see https://github.com/containers/automation_images/pull/149

File: contrib/cirrus/setup.sh
Comment (lsm5) [reply contrib/cirrus/setup.sh:23]: will do

File: contrib/cirrus/setup.sh
Comment (lsm5) [reply contrib/cirrus/setup.sh:26]: yes

Comment (lsm5) [reply contrib/cirrus/setup.sh:26]: I left a few comments on the automation_images PR. PTAL there

File: .cirrus.yml
Comment (lsm5) [reply .cirrus.yml]: done

File: contrib/cirrus/setup.sh
Comment (lsm5) [reply contrib/cirrus/setup.sh]: done

File: contrib/cirrus/runner.sh
Comment (cevich) [inline contrib/cirrus/runner.sh:87]: Minor nit: If these lines are going to stay for now, it might be nice to add a comment pointing maintainers at https://github.com/containers/automation_images/issues/150

Comment (lsm5) [reply contrib/cirrus/runner.sh:90]: done. PTAL

File: contrib/cirrus/setup.sh
Comment (cevich) [inline contrib/cirrus/setup.sh:25]: I don't understand this...won't it always be true?

File: contrib/cirrus/runner.sh
Comment (cevich) [inline contrib/cirrus/runner.sh:90]: I noticed in the Makefile for the old cross target, we were doing a:

`rustup target add aarch64-unknown-linux-gnu`, maybe that's what we're missing for arm?

File: contrib/cirrus/setup.sh
Comment (lsm5) [reply contrib/cirrus/setup.sh:25]: ugh lol i forgot to add the equal aarch64. can't wait for the weekend

File: .cirrus.yml
Comment (cevich) [inline .cirrus.yml:24]: OH! This is the wrong image!  [ami-02da46d8fa51e5316 is the one for podman.  ](https://cirrus-ci.com/task/5118432189349888).  I found this by looking up the image in EC2 and noticed the `Name` tag "fedora-podman-aws-arm64-c5353795591864320".  This is why you're having packaging and compiler issues.  The image you want is `ami-00d7ccde5eaacc50e` which is for `fedora-netavark-aws-arm64` :grin:

Comment (cevich) [reply .cirrus.yml:24]: Ugh, this is horrible.  I really need to do something to make that less confusing.  Otherwise every single `ami-*` looks just like every other `ami-*` :cry:   Why won't AWS just let us use the damn name!

File: .cirrus.yml
Comment (cevich) [reply .cirrus.yml:96]: Maybe update the comment though to delete "cross-".

Comment (lsm5) [reply .cirrus.yml:96]: done

**Pull request merged**
